client/lib/plans/constants.js

```diff
@@ -1329,14 +1343,18 @@ export const FEATURES_LIST = {
 
 	[ FEATURE_CONCIERGE_SETUP ]: {
 		getSlug: () => FEATURE_CONCIERGE_SETUP,
-		getTitle: () => i18n.translate( 'Concierge Setup' ),
-		hideInfoPopover: true,
+		getTitle: () => i18n.translate( 'Jetpack Concierge' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Get help from a Jetpack Concierge' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).